sequencing-server/jestGlobalSetup.js

```diff
@@ -4,7 +4,8 @@ config();
 
 process.env.MERLIN_GATEWAY_URL = process.env.MERLIN_GATEWAY_URL ?? 'http://localhost:9000';
 process.env.MERLIN_GRAPHQL_URL = process.env.MERLIN_GRAPHQL_URL ?? 'http://localhost:8080/v1/graphql';
-process.env.HASURA_GRAPHQL_ADMIN_SECRET = process.env.HASURA_GRAPHQL_ADMIN_SECRET ?? '';
+process.env.HASURA_GRAPHQL_JWT_SECRET = process.env.HASURA_GRAPHQL_JWT_SECRET ?? '{ "type": "HS256", "key": "oursupersecretsupersecurekey1234567890" }'
+process.env.HASURA_GRAPHQL_ADMIN_SECRET = process.env.HASURA_GRAPHQL_ADMIN_SECRET ?? 'aerie';
```

Comment:
I'm not sure if we want to be hardcoding the dev env variables, shouldn't we just require that the user set them?

Reply:
Yeah we should not include any hardcoded secrets.